### PR TITLE
Code: codeownership for feature toggles v2

### PR DESCRIPTION
**Pull request by ArturWierzbicki**
alternative approach to https://github.com/grafana/grafana/pull/64056

### CODE REVIEW

**Review by bergquist — APPROVED** [review]
LGTM

lets get it merged and poke folks about reviwing the toggles.

File: pkg/services/featuremgmt/toggles_gen_test.go [hunk 1]
```diff
@@ -41,6 +41,61 @@ func TestFeatureToggleFiles(t *testing.T) {
 		}
 	})
 
+	ownerlessFeatures := map[string]bool{
+		"alertingBigTransactions":           true,
+		"trimDefaults":                      true,
+		"disableEnvelopeEncryption":         true,
+		"database_metrics":                  true,
+		"prometheusAzureOverrideAudience":   true,
+		"publicDashboards":                  true,
```
Comment (tolzhabayev) [inline pkg/services/featuremgmt/toggles_gen_test.go:50]: isn't this one and the next one `grafanaDashboardsSquad` @natellium?

Comment (natellium) [reply pkg/services/featuremgmt/toggles_gen_test.go:50]: Yes, this one and `publicDashboardsEmailSharing`

**Review by mildwonkey — APPROVED** [review]
i like it!

File: pkg/services/featuremgmt/toggles_gen_test.go [hunk 2]
```diff
@@ -41,6 +41,69 @@ func TestFeatureToggleFiles(t *testing.T) {
...
+		"alertingBacktesting":               true,
+		"alertingNoNormalState":             true,
+		"logsSampleInExplore":               true,
+		"logsContextDatasourceUi":           true,
+		"lokiQuerySplitting":                true,
+		"individualCookiePreferences":       true,
+		"traceqlSearch":                     true,
+	}
+
+	t.Run("all new features should have an owner", func(t *testing.T) {
```
Comment (ArturWierzbicki) [inline pkg/services/featuremgmt/toggles_gen_test.go:87]: these two tests are inspired by https://sirupsen.com/shitlists

**Comment by DanCech** [comment]
nice!

**Pull request merged**